Analysis/Tasks/PWGHF/HFCandidateCreator3Prong.cxx

```diff
@@ -239,6 +248,15 @@ struct HFCandidateCreator3ProngMC {
         }
       }
 
+      // Ξc± → p± K∓ π±
+      if (flag == 0) {
+        //Printf("Checking Ξc± → p± K∓ π±");
+        auto indexRecXic = RecoDecay::isMatchedMCGen(particlesMC, particle, 4232, array{+kProton, -kKPlus, +kPiPlus}, true, &sign);
+        if (indexRecXic > -1) {
```

Comment:
This is wrong. Check how it is done for Λc±.

Reply:
Sorry, my implementation was probably a refuse from the `RecoDecay::getMatchedMCRec` call. I modified the name of the returned value, as well as the condition inside the `if` statement.